app/src/main/java/de/westnordost/streetcomplete/quests/sidewalk/AddSidewalk.kt

```diff
@@ -75,11 +75,14 @@ class AddSidewalk : OsmElementQuestType<LeftAndRightSidewalk> {
         return roadsWithMissingSidewalks + roadsWithInvalidSidewalkTags
     }
 
+    /* Calculate when footway is too far away from the road to be considered its sidewalk.
+       It is an estimate, and we deliberately err on the side of showing the quest too often. */
     private fun getMinDistanceToWays(tags: Map<String, String>): Float =
         (
             (estimateRoadwayWidth(tags) ?: guessRoadwayWidth(tags)) +
             (estimateParkingOffRoadWidth(tags) ?: 0f) +
-            (estimateCycleTrackWidth(tags) ?: 0f)
+            (estimateCycleTrackWidth(tags) ?: 0f) +
+            1f    // absolute minimum legal sidewalk width is usually above 1.5m, but be even more pessimistic
```

Comment:
```suggestion
            1.5f    // assumed sidewalk width
```
let's assume 1.5m then...